src/Parser/Parser.php

```diff
@@ -160,10 +160,15 @@ private function parseClassLike(
                     // Because the property type is not supported.
                     continue;
                 }
-                $propertyNameToken = $this->tokens->nextElementValueToken();
+
+                $propertyNameValues = [$this->tokens->nextElementValueToken()];
```

Comment:
The contents of the array may need to be changed to match `<int, string>` as follows:)
```suggestion
                $propertyNameValues = [$this->tokens->nextElementValueToken()->getValue()];
```

Reply:
Sorry, I had it differently in my local copy with an array_map but didn't like it and apparently copied over wrong.